docs/en/connector/sink/Elasticsearch.mdx

```diff
@@ -12,7 +12,7 @@ Output data to `Elasticsearch`.
 Engine Supported and plugin name
 
 * [x] Spark: Elasticsearch(supported `ElasticSearch version is >= 2.x and <7.0.0`)
-* [x] Flink: Elasticsearch(supported `ElasticSearch version is >= 2.x and <8.0.0`)
+* [x] Flink: Elasticsearch(supported `ElasticSearch version = 7.x`)
 
```

Comment:
Do we need to state here, if users use 6, they can pack it themselves (and the specific steps)

Reply:
Done

Reply:
Done